en/docs/Learn/ConsumeAPI/GeneratingSDKs/generate-sdks-in-dev-portal.md

```diff
@@ -0,0 +1,42 @@
+#Generate SDKs in Developer Portal
+
+A software development kit (SDK) is a set of software development tools that allows to create applications for a specific platform. If an API consumer wants to create an application, they can generate a client side SDK for a supported language/framework and use it to write a software application to consume the subscribed APIs. 
+
+## Download SDK from Dev Portal
```

Comment:
Shall we use Developer Portal instead of Dev Portal?

Reply:
Fixed